theme/static/style/style.css

```diff
@@ -248,8 +248,7 @@ pre code {
 
 /* Screenshots Section */
 
-/* Title */
-.screenshot-title {
+.screenshot-title{
```

Comment:
Don't do this, we prefer having a space before {, it's that some of the blocks are missing it, but this PR purpose is not the fix the styles we don't care about.